armi/reactor/converters/tests/test_uniformMesh.py

```diff
@@ -62,6 +62,24 @@ def test_computeAverageAxialMesh(self):
         self.assertEqual(refMesh[0], avgMesh[0])
         self.assertNotEqual(refMesh[4], avgMesh[4], "Not equal above the fuel.")
 
+    def test_blueprintCopy(self):
+        """ensure that necessary blueprint attributes are set
```

Comment:
Capitalize the docstring

Reply:
addressed

https://github.com/terrapower/armi/pull/645/commits/834c69c756f09ad130fdc094934c6f7eab874eaa